LibreNMS/OS/Routeros.php

```diff
@@ -170,10 +172,10 @@ public function discoverWirelessRate()
             $sensors[] = new WirelessSensor(
                 'rate',
                 $this->getDeviceId(),
-                '.1.3.6.1.4.1.14988.1.1.1.8.1.13.' . $index,
```

Comment:
Is this definitely supposed to be moved?

Reply:
Yep. Last time I put the wrong OID into the 60g rate. That and the fact that the station end, the index seems to be 4 not 1, meant I had to get it to look at the station table as well. Initially the raw rate and the distance values   werealmost identical mesurements 23m and 2.3gigbit which is what made me think it was all fine.

![image](https://user-images.githubusercontent.com/11468979/47883478-120e3780-de7d-11e8-98d9-72a675df0340.png)

And for the normal wireless. 
![image](https://user-images.githubusercontent.com/11468979/47883511-2f430600-de7d-11e8-9afa-c46a9aad013a.png)
![image](https://user-images.githubusercontent.com/11468979/47883518-38cc6e00-de7d-11e8-8b7f-09b94cd4c310.png)